src/com/amazon/ion/impl/lite/IonValueLite.java

```diff
@@ -964,8 +964,32 @@ public void makeReadOnly()
 
     void makeReadOnlyInternal()
     {
-        clearSymbolIDValues();
-        _isLocked(true);
+        IonContainerLite.SequenceContentIterator[] iteratorStack = new IonContainerLite.SequenceContentIterator[CONTAINER_STACK_INITIAL_CAPACITY];
+        int iteratorStackIndex = -1;
+        IonContainerLite.SequenceContentIterator currentIterator = null;
+        IonValueLite value = this;
+        do {
+            if (!(value instanceof IonContainerLite)) {
+                clearSymbolIDValues();
+                _isLocked(true);
```

Comment:
Doesn't this need to be done for the container itself as well? See lines 477-478 of IonContainerLite (removed in this PR).

If so, then I don't understand why the tests still pass.

Reply:
This was missed when I modified local changes to commit. Let me add it back here. 
But it seems like the workflow didn't trigger to run tests for this PR.